### PR TITLE
[merged] remove redundant calls to clean_config_files

### PR DESCRIPTION
**Pull request by stephen679**
Added back in clean_config_files that I accidentally deleted in previous merge.

### CODE REVIEW

**Comment by rhvgoyal** [comment]
@stephen679 I am rethinking. Tests are passing even without clean_config_files(), so may be it is redundant. 

In fact we call cleanup() after test finishes and that calls clean_config_files(). So new test should get a clean slate. And run-tests.sh makes sure config files are not present before running first test.

So may be this clean_config_file() in the middle of test is redundant. 

Lets leave it for now as it is. If we run into issues we will introduce it again.

Test1 and Test6, still seem to be cleaning files, can you remove that call from those tests as well.

**Comment by stephen679** [comment]
@rhvgoyal I cleaned up the files, PTAL.

**Comment by rhvgoyal** [comment]
Can you merge both the patches into one and push again.

File: tests/001-test-use-devs-to-create-thin-pool.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ source $SRCDIR/libtest.sh
 # Test DEVS= directive. Returns 0 on success and 1 on failure.
 test_devs() {
   local devs=$TEST_DEVS
-  local test_status
+  local test_status=1
```
Comment (rhvgoyal) [inline tests/001-test-use-devs-to-create-thin-pool.sh]: I think making test_status related cleanup is a good idea but you seem to be making only for 1 test while others test seem to have the same thing.

Whatever change you do, please keep it same for all tests.

I would suggest that remove test_status related change from this PR and generate a separate PR for all test_status related cleanup for all tests.

File: tests/003-test-override-signature-wipes-existing-signatures.sh [hunk 2]
```diff
@@ -12,6 +12,7 @@ test_override_signatures() {
     return 1
   fi 
 
+  # Create config file
```
Comment (rhvgoyal) [inline tests/003-test-override-signature-wipes-existing-signatures.sh]: There is no need to keep comments consistent across all tests.

File: tests/004-test-non-absolute-disk-name-support.sh [hunk 3]
```diff
@@ -18,7 +18,8 @@ test_non_absolute_disk_name() {
     echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
     return 1
   fi
- 
+
+  # Create config files
```
Comment (rhvgoyal) [inline tests/004-test-non-absolute-disk-name-support.sh]: No need to keep comments consistent across all tests.

File: tests/002-test-reject-disk-with-lvm-signature.sh [hunk 4]
```diff
@@ -9,11 +9,12 @@ test_lvm_sig() {
   local vg_name="dss-test-foo"
 
   # Error out if any pre-existing volume group vg named dss-test-foo
-  if vg_exists "vg_name"; then
+  if vg_exists "$vg_name"; then
```
Comment (rhvgoyal) [inline tests/002-test-reject-disk-with-lvm-signature.sh]: All related changes are generally kept in one patch.

So one good way to organize this patch series would be.

- First patch fixes this vg_name issue
- Second patch fixes cleanup config file issue across remaining tests.
- Third patch fixes test_status issue across all tests.

**Comment by rhvgoyal** [comment]
LGTM. Thanks.

**Comment by rhvgoyal** [comment]
@cgwalters-bot r+

**Comment by rhvgoyal** [comment]
@rh-atomic-bot r+

**Comment by rhvgoyal** [comment]
@cgwalters I see author name as "stephen679 <schung@redhat.com>" and not "Stephen Chung <schung@redhat.com>" in commits. Is it because of bot?

**Comment by cgwalters** [comment]
The bot will use whatever is in the original git commit.

**Comment by rhvgoyal** [comment]
@cgwalters ok, thanks. I think he needs to fix author name in his git settings.

**Pull request closed without merging**